facefusion/uis/choices.py

```diff
@@ -3,5 +3,5 @@
 from facefusion.uis.typing import WebcamMode
 
 common_options : List[str] = [ 'keep-temp', 'skip-audio', 'skip-download' ]
-webcam_modes : List[WebcamMode] = [ 'inline', 'udp', 'v4l2' ]
+webcam_modes : List[WebcamMode] = [ 'inline', 'udp', 'v4l2', 'ip_camera' ]
```

Comment:
Modes are defined by protocol name, not by device name
`[ 'inline', 'rtsp', 'udp', 'v4l2' ]`